api/api-doc/storage_service.json

```diff
@@ -1245,7 +1245,7 @@
                   },
                   {
                      "name":"ignore_nodes",
-                     "description":"List of dead nodes to ingore in removenode operation",
+                     "description":"Comma-separated list of dead nodes (host_ids or ip addresses) to ingore in removenode operation",
```

Comment:
What if user passes a mix of host id and ip address, can we detect and reject this?

Reply:
Why do we care?
I think that as a follow-up we should deprecate the use of ip-addresses for ignore nodes, and eventually we'll support only host IDs.

Reply:
This is a user api change. We care because we need to make sure wrong user inputs are handled. Do we handle it correct?

Reply:
Mixing the host id and ip are most probably an operation mistake. I think it is better just to reject such user input. The list has to be either all host id or all ips.